package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "7val",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Sevenval Developer Toolbelt",
+  "main": "index.js",
+  "scripts": {
+    "test": "lerna run test",
+    "lerna": "lerna",
+    "changelog": "lerna-changelog",
+    "lint": "standard --fix",
+    "commitmsg": "commitlint -e $GIT_PARAMS",
+    "commit": "commit",
+    "precommit": "lint-staged"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:7val/7val.git"
+  },
+  "keywords": [
+    "7val",
+    "toolbelt",
+    "cli"
+  ],
+  "author": "",
+  "license": "ISC",
```

Comment:
I'm not sure about this Licence....
this basically means I can give it to anyone in the world